### DISCUSSION TITLE
react-docgen-typescript is a MAJOR bottleneck, storybook 7 worked fine

### DISCUSSION THREAD

**Discussion by viveleroi** [storybookjs/storybook#29021] [Help]
### Summary

After updating from storybook 7.6.7 to storybook 8.2.9, our fast reload times when editing code hit ~20 seconds. It's absolutely impossible to use storybook like this. When trying react-docgen, the issue goes away (although we obviously lose typescript support). 

Performance on v7 was good enough that we never noticed issues, so this is an extreme downgrade. I'm not sure what else I could try or what might the issue. Our typescript configs haven't changed.

### Additional information

Vite: 5.4.0
Node: 21.5.0

**Comment by valentinpalkovic** [comment]
`react-docgen` has Typrscript support, although it is not as fully-fledged as `react-docgen-typescript`
